.circleci/config.yml

```diff
@@ -802,8 +802,8 @@ workflows:
           name: build-redis
           container-image: redis
       - build:
-          name: build-registry
-          container-image: registry
+          name: build-distribution
+          container-image: distribution
```

Comment:
It would be nice to move this block above because `build` seems sorted alphabetically.